### PR TITLE
Prometheus

### PR DESCRIPTION
**Pull request by PavloTarnovetskyi**
What is this change?

changed script according to a recommendation

**Why is this important?**

best practice

**How this has been tested?**

manually

**How can this be rolled back?**

Rollback can be performed by reverting these changes in source control and redeploying via standard deployment mechanisms. This change is safe to roll back.

**Are there any security risks introduced by this change?**

Screenshot 2022-10-04 at 16 24 21

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -62,10 +61,12 @@ pipeline {
         }
         stage('Substitute public ip in prometheus.yml & restart prometheus'){
             steps{
-                sh 'chmod +x ./terraform/script.sh'
-                sh './terraform/script.sh'
+                dir('./terraform'){
+                sh 'chmod +x ./script.sh'
+                sh './script.sh'
```
Comment (nos373737) [inline Jenkinsfile]:
```suggestion
                sh """
                chmod +x ./script.sh
                ./script.sh
                """
```

Comment (nos373737) [reply Jenkinsfile]: Better way is to call shell once instead of calling it two times.

**Comment by nos373737** [comment]
In general this time more better. If you facing some error try your best to find out the root cause of it.

**Comment by PavloTarnovetskyi** [comment]
Thank you for your recommendation. I'll try to do my best.

**Review by nos373737 — APPROVED** [review]
lgtm

**Pull request merged**